src/controllers/userController.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+
+const {
+  getAll,
+  createUser,
+  getOne,
+  deleteOne,
+  updateOne,
+} = require('../services/userServices');
+
+const getAllUsers = async (_, res) => {
+  try {
+    const users2 = await getAll();
+
+    res.send(users2);
+  } catch (e) {
+    res.sendStatus(400);
+  }
+};
+
+const createNewUser = async (req, res) => {
+  const { name } = req.body;
+
+  try {
+    if (!name || typeof name !== 'string') {
+      throw new Error('Bad request');
+    }
```

Comment:
Could you consider moving this logic to middleware? Also, you can use a library for validation of request data